src/test/java/emissary/output/filter/AbstractFilterTest.java

```diff
@@ -0,0 +1,104 @@
+package emissary.output.filter;
+
+import emissary.config.Configurator;
+import emissary.config.ServiceConfigGuide;
+import emissary.core.DataObjectFactory;
+import emissary.core.IBaseDataObject;
+import emissary.test.core.junit5.UnitTest;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class AbstractFilterTest extends UnitTest {
+
+    AbstractFilter getAbstractFilterInstance() {
+        return new AbstractFilter() {
+            @Override
+            public int filter(IBaseDataObject d, Map<String, Object> params) {
+                return 0;
+            }
+        };
+    }
+
+    AbstractFilter getJsonLangDenyFilter() {
+        Configurator config = new ServiceConfigGuide();
+        config.addEntry("OUTPUT_SPEC_FOO", "/tmp/%S%.%F%");
+        config.addEntry("OUTPUT_TYPE", "JSON");
+        config.addEntry("DENYLIST", "JSON.BSON.V_1.*");
+        config.addEntry("DENYLIST", "JSON.JSON_ONE_LINE");
+        config.addEntry("DENYLIST", "JSON.JSON_LANG_*");
```

Comment:
How would output be affected this behave if someone added the following entry?
```
config.addEntry("DENYLIST", "J*");
```

Reply:
We still need to disallow the scenario for which a DENYLIST entry is configured with a partially-wildcarded filetype.  We don't want to allow that configuration.